week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,11 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+function percentage(numberOfMentors, numberOfStudents) {
```

Comment:
Excellent usage of a function! Well done 💪 

Do you think this is the best name for the function? Can you think of a better one?

Reply:
@ivinapontes Nice work

Reply:
Thank you for the feedback @clems27 !

Reply:
Do you think that naming this function getPercentage or getPercent for short would be better? Thank you for your feedback, Ali @40thieves!  I appreciate it!

Reply:
I think `getPercentages` is a good name, but we usually try to name functions after the core of what they do. In this case the function ultimately has 1 thing that it does at the end. It prints out the percentages, so do you think a name would be `printPercentages`?

Reply:
`getPercent` is a reasonable name, but in my experience, most of the times I've tried to shorten a variable/function name it has been worse for readability in the long term.